src/scope-analyzer.js

```diff
@@ -22,8 +22,19 @@ import { DeclarationType } from './declaration';
 import { ScopeType } from './scope';
 import StrictnessReducer from './strictness-reducer';
 
+function asSimpleFunctionDeclarationName(statement) {
+  return statement.type === 'FunctionDeclaration' && !statement.isGenerator && !statement.isAsync
+    ? statement.name
+    : statement.type === 'LabeledStatement'
+      ? asSimpleFunctionDeclarationName(statement.body)
+      : null;
+}
+
 function getFunctionDeclarations(statements) {
```

Comment:
We should give this a more descriptive name.

Reply:
Renamed to `getUnnestedSimpleFunctionDeclarationNames`.